packages/colorpickers/src/styled/ColorPicker/StyledHexField.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+import { Field } from '@zendeskgarden/react-forms';
+
+const COMPONENT_ID = 'colorpickers.colorpicker.hex_field';
+
+export const StyledHexField = styled(Field).attrs({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})`
```

Comment:
Please apply a `'spellcheck': false` attribute to this field. https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/spellcheck